addons/l10n_latam_check/models/l10n_latam_checkbook.py

```diff
@@ -0,0 +1,59 @@
+from odoo import fields, models, api, _
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class L10nLatamCheckbook(models.Model):
+
+    _name = 'l10n_latam.checkbook'
+    _description = 'Checkbook'
+
+    name = fields.Char(compute='_compute_name', store=True)
+    sequence_id = fields.Many2one(
+        'ir.sequence', 'Sequence', copy=False, domain=[('code', '=', 'own_check')], help="Checks numbering sequence.")
+    next_number = fields.Integer(related='sequence_id.number_next_actual', related_sudo=True, readonly=False)
+    type = fields.Selection(
+        [('deferred', 'Deferred'), ('currents', 'Currents'), ('electronic', 'Electronic')],
+        string='Check type', required=True, default='deferred')
+    journal_id = fields.Many2one(
+        'account.journal', 'Journal', readonly=True, required=True, ondelete='cascade',)
+    range_to = fields.Integer(
+        'To Number',
+        help='If you set a number here, this checkbook will be automatically'
+        ' set as used when this number is raised.'
```

Comment:
Where is this done?

Reply:
@jjscarafia So you mean if the check number was already used for example? (that will be a possible check using the check_number)

About the user jumping numbers, indeed is totally possible that for multiple reasons a user will not follow a sequence or have gaps between checks (stolen/lost check, stolen/lost checkbook, void check, etc)
cc' @jco-odoo

Reply:
Done, I've removed that help and added a warning when check number is greater than range_to
![image](https://user-images.githubusercontent.com/3016656/169814661-0359b9ba-d60f-4f23-9447-d81cf4bc11e1.png)